data/data/aws/bootstrap/main.tf

```diff
@@ -39,7 +40,8 @@ resource "aws_s3_bucket_object" "ignition" {
 
 data "ignition_config" "redirect" {
   replace {
-    source = "s3://${aws_s3_bucket.ignition.id}/bootstrap.ign"
+    source = var.ignition_presigned_url
+    //source = "s3://${aws_s3_bucket.ignition.id}/bootstrap.ign"
```

Comment:
No need to keep this comment as we have it in git history.

Reply:
Fixed!